definitions/npm/react-router-dom_v6.x.x/flow_v0.201.x-/react-router-dom_v6.x.x.js

```diff
@@ -226,6 +226,246 @@ declare module 'react-router-dom' {
 
   declare export type InitialEntry = string | Partial<Location>;
 
+  declare type LowerCaseFormMethod =
+    | 'get'
+    | 'post'
+    | 'put'
+    | 'patch'
+    | 'delete';
+  declare type UpperCaseFormMethod =
+    | 'GET'
+    | 'POST'
+    | 'PUT'
+    | 'PATCH'
+    | 'DELETE';
+
+  /**
+   * Users can specify either lowercase or uppercase form methods on <Form>,
+   * useSubmit(), <fetcher.Form>, etc.
+   */
+  declare export type HTMLFormMethod =
+    | LowerCaseFormMethod
+    | UpperCaseFormMethod;
+
+  declare type V7_FormMethod = UpperCaseFormMethod;
+
+  declare export type FormEncType =
+    | 'application/x-www-form-urlencoded'
+    | 'multipart/form-data';
+
+  /**
+   * Potential states for state.navigation
+   */
+  declare export type NavigationStates = {
```

Comment:
Apologies, this should have some exactness to it but there was a bug in our test runner. I've just fixed this for you as part of https://github.com/flow-typed/flow-typed/commit/8ae0b4cb9e6577ece31cebdb8fd6ea9cf43849de. Do you mind pulling from master to fix errors?